src/CommandingOfficers/Qis.java

```diff
@@ -0,0 +1,234 @@
+package CommandingOfficers;
+
+import java.util.ArrayList;
+import Engine.GamePath;
+import Engine.GamePath.PathNode;
+import Engine.GameScenario;
+import Engine.XYCoord;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameEvents.MapChangeEvent;
+import Engine.UnitMods.UnitFightStatModifier;
+import Engine.UnitMods.UnitModifier;
+import Engine.UnitMods.UnitMovementModifier;
+import Terrain.Environment;
+import Terrain.Environment.Weathers;
+import Terrain.MapMaster;
+import Terrain.TerrainType;
+import Units.Unit;
+import Units.UnitContext;
+import Units.MoveTypes.MoveType;
+
+public class Qis extends Commander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Qis");
+      infoPages.add(new InfoPage(
+          "Commander Qis (pronounced 'keese') has spent his adult life fleeing cold weather, and failing to escape it.\n" +
+          "Whether the cause is simply incredible misfortune or some sort of curse is unclear... he is quite curt on the subject.\n" +
+          "What is known is that Qis has become incredibly adept at using the ever-present snow to his advantage."));
+      infoPages.add(new InfoPage(
+          "Passive:\r\n" +
+          "Watery terrain slows your non-naval units, but all units have perfect movement in snow.\n" +
+          "The path each unit takes on your turn is coated with snow until your next turn."));
+      infoPages.add(new InfoPage(
+          GroovinMovin.NAME+" ("+GroovinMovin.COST+"):\n" +
+          "+"+GroovinMovin.BUFF+" move for all units\n" +
```

Comment:
The '+' is not rendered in the CO description screen. How long has that been broken? It's not just a problem for Qis.

Reply:
We haven't ever had a + symbol, except when using system fonts. I just put the + in there by habit.
...I guess that's in-scope for `presentationFixes`, innit

Reply:
I could swear we had that working at one point. I wouldn't have written a bunch of CO ability descriptions with pluses if it hadn't been working at the time.

Reply:
It was probably working before we had fonts? IDK.
Either way, I fixed it in `presentationFixes`.